### PR TITLE
Addons bucket with Gantt_chart

### PR DESCRIPTION
**Pull request by marcocast**

### CODE REVIEW

**Review by paraita — APPROVED** [review]
There's a commit with 3 times the same comment !

**Review by ShatalovYaroslav — APPROVED** [review]
LGTM

File: src/main/resources/default-workflows/Addons/Gantt_chart.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="Gantt_chart" projectName="Scheduler Analytics"
+    priority="normal"
+    onTaskError="continueJobExecution">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Addons/Gantt_chart.xml]: ctrl-M caracter

**Review by mbenguig — APPROVED** [review]
I dont know why there is a ctrM character which disappears in raw vue.. but it is ok!

**Pull request merged**